test_improve_iti.jl

```diff
@@ -0,0 +1,87 @@
+import Dolo
```

Comment:
Do we ever intend to have CI run these tests? I'm happy if we don't, I just wanted to mention that they aren't being run right now

Reply:
not yet, iti is still *experimental*